clippy_lints/src/unnecessary_wraps.rs

```diff
@@ -110,39 +133,42 @@ impl<'tcx> LateLintPass<'tcx> for UnnecessaryWraps {
         });
 
         if can_sugg && !suggs.is_empty() {
```

Comment:
Maybe I don't understand, but it seems that `can_suggest` will only be true if `!suggest.is_empty()`. So the bool is redundant.

Reply:
Oh I see! Thank you!